app/src/main/java/org/simple/clinic/encounter/sync/EncounterPullResponse.kt

```diff
@@ -0,0 +1,16 @@
+package org.simple.clinic.encounter.sync
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import org.simple.clinic.sync.DataPullResponse
+
+@JsonClass(generateAdapter = true)
+data class EncounterPullResponse(
+
+    @Json(name = "encounters")
+    override val payloads: List<EncounterPayload>,
+
+    @Json(name = "process_token")
+    override val processToken: String
```

Comment:
Curious, what is a process token?

Reply:
Process token is an encoded string that has all the information related to batch syncing like last synced timestamp. This token is set by the server and we save it and send it across the next time we pull records from server. The server then send us records that were updated post the last sync.
There is more information in that token which is useful to the server mostly.

Reply:
FYI, `process_token` is added to all the APIs that involve syncing.